setup.py

```diff
@@ -33,17 +33,17 @@ def read(*rnames):
         'jinja2',
         'packaging',
         'pygments',
+        'pyyaml',
         'reportlab',
         'smartypants',
-        'pyyaml',
     ],
     extras_require={
-        'tests': ['pyPdf2', 'pymupdf'],
-        'sphinx': ['sphinx'],
-        'svgsupport': ['svglib'],
         'aafiguresupport': ['aafigure>=0.4'],
```

Comment:
Do we still need this version constraint? requirements.in does not have this.

Reply:
I would think we can remove this given that 0.4 came out in 2009, so I can't imagine that many systems that have been upgraded to Py3 continue to have 0.3 installed!

Reply:
Removed in cd051a277209c8a5ec6df3108d61bbcab2d208e4.